lib/api/element-commands.js

```diff
@@ -340,8 +340,13 @@ module.exports = function(client) {
   function CommandAction(using, value, protocolAction, args, callback, originalStackTrace) {
     events.EventEmitter.call(this);
 
-    var $this = this;
-    var el = Protocol.element(using, value, function(result) {
+    var self = this;
+    var el = Protocol.elements(using, value, function(result) {
+
+      if (result.status === 0 && result.value.length === 0) {
```

Comment:
This independent check handles the setting of status to -1 which is needed for the callback to know that there was a failure in the request.  `elements()` returning 0 elements is not a failure status on its own, but we expect at least one element to be found, so in that case we flip the flag for status to mark it as a failure as we expect at least one to be found (as per original `element()` behavior)